scripts/scripts.js

```diff
@@ -427,6 +427,23 @@ function decorateLinks(main) {
     }
   });
 }
+
+function decorateImages(main) {
+  // Get all img elements within the main container
+  const images = main.querySelectorAll('img');
+
+  // Get the first image and set class to hero-image
+  const firstImage = images[0];
+  if (firstImage) {
+    firstImage.classList.add('hero-image');
+  }
+
+  // Loop through the rest of the images and set class to article-image
+  for (let i = 1; i < images.length; i += 1) {
+    images[i].classList.add('article-image');
```

Comment:
you can start at 0 here and then in css you can do:

```css
.default-content-wrapper .article-image {
  margin-top: 2.4re;
  margin-bottom: 2.4rem;
}

.default-content-wrapper .article-image.hero-image {
  margin-top: 0;
}
```

Reply:
Shouldn't I add the margin-bottom: 2.4rem in the hero image as well?

Reply:
@sunidhiazad if you start at the index 0, the `article-image` will be added to the hero too, so the first rule will apply margin bottom to the hero as well. the second rule would simply remove the margin top.